cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsInstanceConnector.java

```diff
@@ -131,25 +127,31 @@ public String getConsoleOutput(AuthenticatedContext authenticatedContext, CloudI
     )
     @Override
     public List<CloudVmInstanceStatus> reboot(AuthenticatedContext ac, List<CloudInstance> vms) {
-        AmazonEC2Client amazonEC2Client = awsClient.createAccess(new AwsCredentialView(ac.getCloudCredential()),
-                ac.getCloudContext().getLocation().getRegion().value());
-        List<CloudInstance> affectedVms = new ArrayList<>();
+        List<CloudVmInstanceStatus> rebootedVmsStatus = new ArrayList<>();
```

Comment:
one more nit, but we can merge it without even if you don't change this:
you can declare this variable right in the line where it gets value, at `doStart`. also the return could be moved inside the try, right after `logInvalidStatuses(getNotStarted(statuses), InstanceStatus.STARTED);`

Reply:
oh I missed `if (!vms.isEmpty())`, nevermind....